src/UuidFactory.php

```diff
@@ -55,7 +55,7 @@ class UuidFactory implements UuidFactoryInterface
      */
     private $timeGenerator = null;
 
-    /**
+    /*
```

Comment:
I wasn't sure why this happened either, but I fixed it in https://github.com/ramsey/uuid/commit/d88a941b8c7b7618fcf1f3c41eaa0deb14cadfea. :-)